docs/docs/segment-path.md

```diff
@@ -37,6 +37,7 @@ Display the current path.
 is set to `true`)
 - mapped_locations_enabled: `boolean` - replace known locations in the path with the replacements before applying the
 style. defaults to `true`
+- enable_hyperlink: `boolean` - displays an hyperlink for the path - defaults to `false`
```

Comment:
Can't believe I missed this, do we want to keep it at this level or higher up at `Settings`?

Reply:
Now, the engine detects url from any segments. Hyperlink can be enabled/disabled for each segment and it takes care of rendering the text or the url markdown.  
On the other side, it could be enabled at global level and implying that all segments supporting url will render them.
Both have their pros and cons.  
It’s a matter of taste :).

Reply:
I believe both can be an option, but let's keep it like this for now. Amazing work.